app/src/main/java/org/fossasia/openevent/general/utils/DataBindingAdapters.kt

```diff
@@ -73,24 +75,39 @@ fun setExpired(imageView: ImageView, isExpired: Boolean) {
 
 @BindingAdapter("orderStatus")
 fun setOrderStatus(textView: TextView, orderStatus: String) {
+    val resource = Resource()
     when (orderStatus) {
         ORDER_STATUS_PLACED -> {
             textView.isVisible = true
             textView.text = ORDER_STATUS_PLACED
-            textView.setBackgroundResource(R.color.orderStatusBlue)
+            resource.getColor(R.color.orderStatusBlue)?.let {
+                textView.background.setColorFilter(it, PorterDuff.Mode.SRC_ATOP)
+            }
         }
         ORDER_STATUS_COMPLETED -> {
             textView.isVisible = true
             textView.text = ORDER_STATUS_COMPLETED
-            textView.setBackgroundResource(R.color.orderStatusGreen)
+            resource.getColor(R.color.orderStatusGreen)?.let {
+                textView.background.setColorFilter(it, PorterDuff.Mode.SRC_ATOP)
+            }
         }
         ORDER_STATUS_PENDING -> {
             textView.isVisible = true
             textView.text = ORDER_STATUS_PENDING
-            textView.setBackgroundResource(R.color.orderStatusOrange)
+            resource.getColor(R.color.orderStatusOrange)?.let {
+                textView.background.setColorFilter(it, PorterDuff.Mode.SRC_ATOP)
+            }
         }
         else -> {
             textView.isVisible = false
         }
     }
 }
+
+@BindingAdapter("ticketNumber")
+fun setTicketNumber(textView: TextView, ticketNumber: Int) {
+    if (ticketNumber == 1)
+        textView.text = "See 1 ticket"
+    else
+        textView.text = "See $ticketNumber tickets"
```

Comment:
@iamareebjamal in here, when I use resources.getQuantityString from Resource file, it always return something like this "[Ljava.lang.Object;@88553da ticket". What should I do then or should I keep it this way?

Reply:
For now make it ticket(s)